.github/workflows/check-code-quality.yml

```diff
@@ -4,14 +4,15 @@ on:
     branches: [ "master" ]
   pull_request:
     branches: [ "master" ]
+    types: ["labeled", "opened", "synchronize", "reopened"]
```

Comment:
See this for explanations of these: https://frontside.com/blog/2020-05-26-github-actions-pull_request/